MiniApp/Classes/admob/AdMobDisplayer.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import GoogleMobileAds
+
+/// This subclass of [MiniAppAdDisplayer](x-source-tag://MiniAppAdDisplayer) is internally used by Mini App SDK to display Google Ads
+internal class AdMobDisplayer: MiniAppAdDisplayer {
+    var interstitialAds: [String: GADInterstitial] = [:]
+    var rewardedAds: [String: GADRewardedAd] = [:]
+    var onInterstitialLoaded: [String: (Result<Void, Error>) -> Void] = [:]
+    var admobStartResult: [String : GADAdapterStatus]?
+
+    override init() {
+        super.init()
+        delegate = self
+    }
+
+    override func initFramework() {
+        GADMobileAds.sharedInstance().start {[weak self] status in
```

Comment:
I think it should be the host app's responsibility to initialize the Ad Mob SDK. For example, they might also be using Ad Mob somewhere else in their App so they would want to initialize it at App launch.